app/controllers/services/service_instances_controller.rb

```diff
@@ -347,6 +348,12 @@ def unbind_route(route_guid, instance_guid)
 
     private
 
+    class ServiceInstanceSharedToEagerLoader
```

Comment:
This class will not be private unless you do 
```rb
private_constant :ServiceInstanceSharedToEagerLoader
```

Reply:
Thanks, we've made this change.

Reply:
Changes for this included in #1033 and can be viewed here: https://github.com/cloudfoundry-incubator/cloud_controller_ng_sapi/commit/ff8276e5d201e573df03640a0f6865288b2d55af#diff-9b86a08059c9bdc2b6875a70abf9ee29R366